piet-common/src/direct2d_back.rs

```diff
@@ -165,30 +110,28 @@ impl Device {
 
     /// Create a new bitmap target.
     pub fn bitmap_target(
-        &self,
+        &mut self, //TODO this changed to mut, double check that this it ok
```

Comment:
I think so. There's a tendency for wrapped com to be `&self` with interior mutability, but that's because of technical details. This feels like a case where `&mut` is the right type, but if doc for the underlying device was explicitly marked as thread-safe, we might relax it.

(It's worth noting that taking a `&self` and calling a non-thread-safe function is a potential safety violation. So `&mut self` is always more conservative if you're not sure)

Reply:
resolved, had to also update cairo backend's `bitmap_target` to take `&mut self`, as well as png example.